### PR TITLE
UI

### PR DESCRIPTION
**Pull request by tetuaoro**
- [x] Setup page
- [x] Login page
- [ ] Admin page *(me.vue)*

### CODE REVIEW

**Comment by tetuaoro** [comment]
wow build step is fast

File: src/app/pages/me.vue
Comment (tetuaoro) [inline src/app/pages/me.vue:1]: I'll update `components` later as I didn't see that `radix-vue` had been added.

Comment (kaaax0815) [reply src/app/pages/me.vue:1]: Could u also make this pr against https://github.com/kaaax0815/wg-easy/tree/feat-admin

Comment (tetuaoro) [reply src/app/pages/me.vue:1]: Can't find your repo

image

Comment (kaaax0815) [reply src/app/pages/me.vue:1]: You can create it here https://github.com/kaaax0815/wg-easy/compare/feat-admin...tetuaoro:wg-easy:admin-ui

File: src/app/components/error/Toast.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@ const props = defineProps<{
 
 const open = ref(true);
 
-const autoCloseToast = props.duration ? Number(props.duration) : 5000; // 5 seconds
+const autoCloseToast = props.duration ? Number(props.duration) : 12000; // 12 seconds
```
Comment (kaaax0815) [inline src/app/components/error/Toast.vue:10]: https://www.radix-vue.com/components/toast#custom-duration

**Review by kaaax0815 — CHANGES_REQUESTED** [review]

File: src/app/components/error/Toast.vue (hunk 1 above)
Comment (tetuaoro) [reply src/app/components/error/Toast.vue:10]: thx

**Comment by kaaax0815** [comment]
Handled in https://github.com/kaaax0815/wg-easy/pull/4

**Pull request closed without merging**